src/plugins/condition/criterion/TelemetryCriterion.js

```diff
@@ -0,0 +1,85 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2019, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+import * as EventEmitter from 'eventemitter3';
+
+export default class TelemetryCriterion extends EventEmitter {
+
+    constructor(telemetryDomainObjectKey, openmct) {
+        super();
+
+        this.openmct = openmct;
+        this.telemetryObject = this.openmct.objects.get(this.openmct.objects.makeKeyString(telemetryDomainObjectKey));
+        this.telemetryAPI = this.openmct.telemetry;
+        this.subscription = null;
+        this.telemetryMetadata = null;
```

Comment:
This variable seems to be unused.

Reply:
Yes, but it will be later.

Reply:
It should be commented out then if you think it's definitely going to be used later and want to keep it. I understand this is a work in progress, but it's better to keep the code clean.